module/.cypress/plugins/index.js

```diff
@@ -0,0 +1,13 @@
+const injectDevServer = require('@cypress/react/plugins/load-webpack');
```

Comment:
is this affected by vite?

Reply:
Yes, but the good news is that when Armstrong moves to Vite there's already a 'how to move Cypress to Vite' guide available.